### ISSUE TITLE
RSS feed for the aggregate

### ISSUE THREAD

**Issue by audreywatters** [indieedtech/rss-aggregator#3]
Is it possible to add an RSS feed for this aggregation?

**Comment by adamcroom** [comment]
Yes, @kinlane can confirm, but I believe you just add the blog titles and URLs here: https://github.com/indieedtech/rss-aggregator/blob/gh-pages/_data/feed-urls.json

**Comment by kinlane** [comment]
I added it - http://indieedtech.github.io/rss-aggregator/aggregate.xml - not sure where I will link up on main site yet.
